README.md

```diff
@@ -4,6 +4,9 @@
 We will be querying the FoodData Central API with an ingredient to retrieve a list of the ten most relevant foods that include that ingredient.
 
 First, sign up for an API key here: https://fdc.nal.usda.gov/api-key-signup.html
+My API key: yhDafOodE0ip7xhWg4bUbppWfvaPoALxv340AFLh
```

Comment:
Don't include this in your readme :) I suggest copying it there but then removing it before you commit, for future reference. 
(Personally I store my API keys in a password manager if you use one - not necessary but can be handy if you go back to this API later for any personal projects!)